sql/driver/sender.go

```diff
@@ -43,7 +43,7 @@ var defaultRetryOptions = retry.Options{
 type Sender interface {
 	// Send invokes the Call.Args.Method with Call.Args and sets the result
 	// in Call.Reply.
-	Send(context.Context, sqlwire.Call)
+	Send(sqlwire.Request) (sqlwire.Response, error)
```

Comment:
@tschottdorf might want the `context.Context` parameter to be kept. Though if it isn't currently being used I'm fine with removing it.